requirements.txt

```diff
@@ -1,13 +0,0 @@
-certifi==2017.7.27.1
-chardet==3.0.4
-idna==2.6
-oauthlib==2.0.2
-olefile==0.44
-Pillow==4.2.1
-py==1.4.34
-pytest==3.2.1
-requests==2.18.4
-requests-oauthlib==0.8.0
-twython==3.5.0
-urllib3==1.22
-wordnik==2.1.3
```

Comment:
Are you still planning on using the Wordnik package, or are you going to use the API instead? Seems like the package isn't currently maintained.